test/emmy/examples/double_pendulum_test.cljc

```diff
@@ -60,81 +70,95 @@
       (is (= 4 (count @o))))))
 
 (deftest infix-forms
-  (let [eq (e/simplify
-            ((double/state-derivative 'm1 'm2 'l1 'l2 'g)
-             (up 't (up 'theta 'phi) (up 'thetadot 'phidot))))]
-    (is (= (str "function(t, theta, phi, thetadot, phidot) {\n"
-                "  var _0001 = - theta;\n"
-                "  var _0005 = Math.pow(phidot, 2);\n"
-                "  var _0006 = Math.pow(thetadot, 2);\n"
-                "  var _0008 = Math.sin(phi);\n"
-                "  var _0009 = Math.sin(theta);\n"
-                "  var _000a = phi + _0001;\n"
-                "  var _000e = Math.cos(_000a);\n"
-                "  var _0010 = Math.sin(_000a);\n"
-                "  var _0011 = Math.pow(_0010, 2);\n"
-                "  return [1, [thetadot, phidot], [(l1 * m2 * _0006 * _000e * _0010 + l2 * m2 * _0005 * _0010 + g * m2 * _0008 * _000e - g * m1 * _0009 - g * m2 * _0009) / (l1 * m2 * _0011 + l1 * m1), (- l2 * m2 * _0005 * _000e * _0010 - l1 * m1 * _0006 * _0010 - l1 * m2 * _0006 * _0010 + g * m1 * _0009 * _000e + g * m2 * _0009 * _000e - g * m1 * _0008 - g * m2 * _0008) / (l2 * m2 * _0011 + l2 * m1)]];\n"
-                "}")
-           (e/->JavaScript eq
-                           :parameter-order '[t theta phi thetadot phidot]
-                           :deterministic? true))))
+  (is (=
+       ["[y01, [y02, y03], [y04, y05]]"
```

Comment:
does this end up having different performance than manually generating the bindings, or are they equivalent in performance? It would be nice if they were equivalent...